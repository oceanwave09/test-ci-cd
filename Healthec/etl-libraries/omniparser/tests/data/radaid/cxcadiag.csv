batch_type,First Name,Last Name,MRN/ID,DIAG_DATE,DIAG_NAME,DIAG_CODE,PATH_RESULT
8,First26351,Last26351,26351,10-15-2023,1,56820,3
8,First26352,Last26352,26352,09-18-2014,2,168406009,1
8,First26353,Last26353,26353,12-15-2001,3,168407000,5    