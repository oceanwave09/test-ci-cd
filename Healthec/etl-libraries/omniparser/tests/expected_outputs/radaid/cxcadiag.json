[
	{
		"batch_type": "8",
		"diag_code": "56820",
		"diag_date": "10-15-2023",
		"diag_name": "1",
		"diag_result": "3",
		"first_name": "9e7a7d800f451bef2a6bb9e2024d7569",
		"last_name": "74967d45b9d2cfa01bd08352d5ae7802",
		"mrn": "1c26ac8bfc15d08afbccac2ab1ec40fd",
		"row_id": "693b381a-ee9a-4ffc-aaa4-68021bb19751"
	},
	{
		"batch_type": "8",
		"diag_code": "168406009",
		"diag_date": "09-18-2014",
		"diag_name": "2",
		"diag_result": "1",
		"first_name": "8f5a96dc34bd18e95f0fae1ef8328649",
		"last_name": "137fa153590188b32101b22447adbff0",
		"mrn": "3d7c0ad3e51ca2d9fa55d2c374bcd17a",
		"row_id": "dfac25e6-2615-434f-9558-393098cd27c2"
	},
	{
		"batch_type": "8",
		"diag_code": "168407000",
		"diag_date": "12-15-2001",
		"diag_name": "3",
		"diag_result": "5",
		"first_name": "7e0453536700a8d9f19f49441f1a1bab",
		"last_name": "68f3b3f9d7a7f21efee846b1a19cda3b",
		"mrn": "4d94e698dee0e06c55c995dc6f59ea55",
		"row_id": "7664c00c-44dc-4bab-8332-814bb360f060"
	}
]
